src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,44 @@
+.menu {
+  &__list {
+    display: flex;
+    list-style: none;
+    font-weight: 500;
+  }
+
+  &__item {
+    position: relative;
```

Comment:
bottom line should be positioned relatively to link, not the item

Reply:
But then I shoud give display block to links